docs/source/notebooks/ModelBuilder in PyMC-Marketing context.ipynb

```diff
@@ -0,0 +1,1149 @@
+{
```

Comment:
<p>I would show doing posterior predictive on new data after loading the model. That's a more tangible application</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Agreed.</p>

Reply:
<p>Do we have another dataset that could be used to get the new predictions? Otherwise the only approach I see is to use the code that was used to generate this data, and change the random seed, therefore getting slightly different data, is that what you had in mind?</p><p> </p>